### ISSUE TITLE
Stuck IBC transfers

### ISSUE THREAD

**Issue by okannako** [dymensionxyz/roller#559]
### Roller version

v0.1.16-beta

### OS

Ubuntu 20.0 LTS

### Processor Architecture

x86_64

### Steps to reproduce it

IBC Transfer and Faucet

### Expected result

My IBC transfers are not reflected in TVL (Dym)

### Actual result

Also, my dym ibcs have not been reflected on rollApp TVL for three days.

### Relevant log output

```Shell
ts=2023-10-04T22:34:09.080126Z lvl=info msg="Failed to query packet commitments" channel_id=channel-0 port_id=transfer attempt=5 max_attempts=5 error="no error on QueryPacketCommitments for nakoturk_2567-1, however response is nil"
ts=2023-10-04T22:34:09.080148Z lvl=error msg="Failed to query packet commitments after max retries" channel_id=channel-0 port_id=transfer attempts=5 error="no error on QueryPacketCommitments for nakoturk_2567-1, however response is nil"
Failed to execute command: /usr/local/bin/roller_bins/rly, err: fork/exec /usr/local/bin/roller_bins/rly: no such file or directory
ts=2023-10-04T22:34:10.543502Z lvl=info msg="Failed to query packet commitments" channel_id=channel-5254 port_id=transfer attempt=5 max_attempts=5 error="no error on QueryPacketCommitments for froopyland_100-1, however response is nil"
ts=2023-10-04T22:34:10.543524Z lvl=error msg="Failed to query packet commitments after max retries" channel_id=channel-5254 port_id=transfer attempts=5 error="no error on QueryPacketCommitments for froopyland_100-1, however response is nil"
ts=2023-10-04T22:34:10.543553Z lvl=info msg="No packets to relay" src_chain_id=froopyland_100-1 src_port_id=transfer dst_chain_id=nakoturk_2567-1 dst_port_id=transfer
ts=2023-10-04T22:34:11.993049Z lvl=info msg="Error building or broadcasting transaction" provider_type=cosmos chain_id=froopyland_100-1 attempt=1 max_attempts=30 error="rpc error: code = Unknown desc = rpc error: code = Unknown desc = TrustedHeight {1 2563} must be less than header height {1 2563}: invalid header height [cosmos/ibc-go/v5@v5.2.0/modules/light-clients/01-dymint/types/header.go:72] With gas wanted: '0' and gas used: '13368' : unknown request"
ts=2023-10-04T22:34:18.475777Z lvl=info msg="Failed to query packet commitments" channel_id=channel-0 port_id=transfer attempt=1 max_attempts=5 error="no error on QueryPacketCommitments for nakoturk_2567-1, however response is nil"
ts=2023-10-04T22:34:18.770911Z lvl=info msg="Failed to query packet commitments" channel_id=channel-5254 port_id=transfer attempt=1 max_attempts=5 error="no error on QueryPacketCommitments for froopyland_100-1, however response is nil"
ts=2023-10-04T22:34:18.925242Z lvl=info msg="Failed to query packet commitments" channel_id=channel-0 port_id=transfer attempt=2 max_attempts=5 error="no error on QueryPacketCommitments for nakoturk_2567-1, however response is nil"
ts=2023-10-04T22:34:19.572864Z lvl=info msg="Failed to query packet commitments" channel_id=channel-5254 port_id=transfer attempt=2 max_attempts=5 error="no error on QueryPacketCommitments for froopyland_100-1, however response is nil"
ts=2023-10-04T22:34:19.794525Z lvl=info msg="Failed to query packet commitments" channel_id=channel-0 port_id=transfer attempt=3 max_attempts=5 error="no error on QueryPacketCommitments for nakoturk_2567-1, however response is nil"
ts=2023-10-04T22:34:21.482975Z lvl=info msg="Failed to query packet commitments" channel_id=channel-0 port_id=transfer attempt=4 max_attempts=5 error="no error on QueryPacketCommitments for nakoturk_2567-1, however response is nil"
ts=2023-10-04T22:34:21.918346Z lvl=info msg="Failed to query packet commitments" channel_id=channel-5254 port_id=transfer attempt=3 max_attempts=5 error="no error on QueryPacketCommitments for froopyland_100-1, however response is nil"
ts=2023-10-04T22:34:23.919785Z lvl=info msg="Failed to query packet commitments" channel_id=channel-5254 port_id=transfer attempt=4 max_attempts=5 error="no error on QueryPacketCommitments for froopyland_100-1, however response is nil"
ts=2023-10-04T22:34:24.733411Z lvl=info msg="Failed to query packet commitments" channel_id=channel-0 port_id=transfer attempt=5 max_attempts=5 error="no error on QueryPacketCommitments for nakoturk_2567-1, however response is nil"
ts=2023-10-04T22:34:24.733437Z lvl=error msg="Failed to query packet commitments after max retries" channel_id=channel-0 port_id=transfer attempts=5 error="no error on QueryPacketCommitments for nakoturk_2567-1, however response is nil"
ts=2023-10-04T22:34:27.810312Z lvl=info msg="Failed to query packet commitments" channel_id=channel-5254 port_id=transfer attempt=5 max_attempts=5 error="no error on QueryPacketCommitments for froopyland_100-1, however response is nil"
ts=2023-10-04T22:34:27.810338Z lvl=error msg="Failed to query packet commitments after max retries" channel_id=channel-5254 port_id=transfer attempts=5 error="no error on QueryPacketCommitments for froopyland_100-1, however response is nil"
ts=2023-10-04T22:34:27.810399Z lvl=info msg="No packets to relay" src_chain_id=froopyland_100-1 src_port_id=transfer dst_chain_id=nakoturk_2567-1 dst_port_id=transfer
```

### Notes

This problem is a general situation, but let me point out. Facuetten does not send faucet to everyone who wants my token. It goes to one of the five people who want it. The problem may be in the boat.

Also, my dym ibc have not been reflected on rollApp TVL for three days.

**Comment by okannako** [comment]
It's something I've tried many times, but it doesn't work out. This is also the case with many users. I have received a lot of bad feedback from the Turkish community on this issue.

**Comment by mtsitrin** [comment]
@okannako what's the chain id?

the faucet sends the funds on the Froopyland network, from there it needs to be sent with IBC to the rollapp
are ur users did this and it still not working?

**Comment by okannako** [comment]
One of the problems is that faucet does not send tokens to some of them. The other problem is that the ibc transfers made appear days later.

**Comment by omritoptix** [comment]
closing. fixed at release 1.19.
